modules/dxp/apps/osb/osb-site-initializer/osb-site-initializer-partner-portal/extra/remote-app/package.json

```diff
@@ -14,8 +14,9 @@
 		"@clayui/shared": "^3.75.2",
 		"classnames": "^2.3.1",
 		"formik": "^2.2.9",
-		"react": "16.12.0",
-		"react-dom": "16.12.0",
+		"react": "^18.2.0",
+		"react-dom": "^18.2.0",
```

Comment:
@biellopes00 can you remove this change? or rebase with my PR?

https://github.com/pt-liferay-solutions/liferay-portal/pull/2398

I already sent this fix